rpc/src/lib.rs

```diff
@@ -38,3 +38,24 @@ pub mod mayastor {
 
     include!(concat!(env!("OUT_DIR"), "/mayastor.rs"));
 }
+
+#[allow(dead_code)]
+#[allow(clippy::type_complexity)]
+#[allow(clippy::unit_arg)]
+#[allow(clippy::redundant_closure)]
+#[allow(clippy::upper_case_acronyms)]
+pub mod mayastorv1 {
```

Comment:
I'm not sure if the PRs are depending and related on one and other but it would be better here to also use the same approach such that we get `rpc::mayastor::v1`

Reply:
Done.